src/Twilio/Http/CurlClient.php

```diff
@@ -9,26 +9,47 @@
 class CurlClient implements Client {
     public const DEFAULT_TIMEOUT = 60;
     protected $curlOptions = [];
-    protected $debugHttp = false;
 
     public $lastRequest;
     public $lastResponse;
 
     public function __construct(array $options = []) {
         $this->curlOptions = $options;
-        $this->debugHttp = \getenv('DEBUG_HTTP_TRAFFIC') === 'true';
     }
 
     public function request(string $method, string $url,
                             array $params = [], array $data = [], array $headers = [],
                             string $user = null, string $password = null,
-                            int $timeout = null): Response {
+                            int $timeout = null, string $logLevel = null): Response {
         $options = $this->options($method, $url, $params, $data, $headers,
                                   $user, $password, $timeout);
 
         $this->lastRequest = $options;
         $this->lastResponse = null;
 
+        if (getenv('DEBUG_HTTP_TRAFFIC') === 'true') {
+            $logLevel = 'debug';
+        }
+
+        if ($logLevel === 'debug') {
+            \error_log('-- BEGIN Twilio API Request --');
+            \error_log('Request Method: ' . $method);
+            \error_log('Request URL: ' . $url);
+            if (isset($u['query']) && strlen($u['query']) > 0) {
+                $u = \parse_url($url);
+                $hdrLine = $method . ' ' . $u['path'];
```

Comment:
comparing this to the [old implementation](https://github.com/twilio/twilio-php/pull/667/files#diff-f0c91226ea3388e157f785c28ad604c7de97a66e31fd7a988c9aa5e89cabd349L57) it looks like this was only done to log the method and url path together, which is [now done separately](https://github.com/twilio/twilio-php/pull/667/files#diff-f0c91226ea3388e157f785c28ad604c7de97a66e31fd7a988c9aa5e89cabd349R36). I think the parse url should be moved before the `if`, L37 should log just the path portion, and this section should just log `$u['query']`